### ISSUE TITLE
Fine tuning coco evaluation

### ISSUE THREAD

**Issue by gaussiangit** [zylo117/Yet-Another-EfficientDet-Pytorch#347]
I have 5k dataset, I tried fine tuning with D2 to D7 but could not achiever good results.
Total loss did not go below 0.7. Not sure the problem is ! I understand there could be different reasons but anyone faced similar issues with fine tuning or any thoughts ? 

 Following params : --batch_size 32 --lr 1e-5 --num_epochs 500

Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.018
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.057
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.006
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.006
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.019
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.034
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.044
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.096
 Average Rec

**Comment by zylo117** [comment]
probably either overfitting or mislabel
